sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -972,6 +1045,7 @@ case class Logarithm(left: Expression, right: Expression)
   }
 }
 
+
```

Comment:
I'd revert the whitespace changes here, or at least only make changes that make the spacing consistent.